packages/shared/routes/dashboard/staking/views/StakingInfo.svelte

```diff
@@ -108,13 +107,13 @@
             stakingEventState === ParticipationEventState.Commencing
         ) {
             const dateArgument = formatDate(ASSEMBLY_EVENT_START_DATE, { format: 'long' })
-            const localeArguments = { values: { token: Token.IOTA, date: dateArgument } }
+            const localeArguments = { values: { token: 'iota', date: dateArgument } }
```

Comment:
Why not use `Token.IOTA` here?

Reply:
Because this no longer exists, only realised the build failed when fixing previous pr comments. Additionally this file will need to be refactored when brought to v2.